.github/workflows/ci.yml

```diff
@@ -8,8 +8,6 @@ on:
     types:
       - opened
       - synchronize
```

Comment:
Do we also need to include cases where the pull request gets merged with the main branch? Are there other cases we should consider?

Reply:
I think as a start this should cover the cases that we need. We could add the CI to the `main` branch as well, but assuming we go with the flow of opening a pull request and then merging, rather than committing directly to main, this should ensure all the checks are being made before the code touches `main`